README.md

```diff
@@ -32,15 +32,21 @@ Loki differs from Prometheus by focusing on logs instead of metrics, and deliver
 
 * [Installing Loki](https://grafana.com/docs/loki/latest/installation/)
 * [Installing Promtail](https://grafana.com/docs/loki/latest/clients/promtail/installation/)
-* [Getting Started Guide](https://grafana.com/docs/loki/latest/getting-started/)
+* [Getting Started](https://grafana.com/docs/loki/latest/getting-started/)
 
 ## Upgrading
 
 * [Upgrading Loki](https://grafana.com/docs/loki/latest/operations/upgrade/)
 
 ### Documentation
 
-* [master](https://grafana.com/docs/loki/latest/)
+* [v2.2.1](https://grafana.com/docs/loki/latest/)
```

Comment:
Should we link this to `v2.2.1` instead of `latest`?

Reply:
I think we should link directly to `v2.2.1` in the docs and have separate links for `latest` and `next`.

Reply:
Actually, given your comments, I'm going to close this PR.  I will introduce a new PR that revises what we present in the README such that a new release of Loki does not require an update to the README.  That'll reduce the effort of putting out a new release.